vector/src/main/java/im/vector/app/features/home/room/list/actions/RoomListQuickActionsEpoxyController.kt

```diff
@@ -48,54 +46,38 @@ class RoomListQuickActionsEpoxyController @Inject constructor(
         val notificationViewState = state.notificationSettingsViewState
         val roomSummary = notificationViewState.roomSummary() ?: return
         val host = this
-        val isV2 = features.notificationSettingsVersion() == VectorFeatures.NotificationSettingsVersion.V2
-        // V2 always shows full details as we no longer display the sheet from RoomProfile > Notifications
-        val showFull = state.roomListActionsArgs.mode == RoomListActionsArgs.Mode.FULL || isV2
-
-        if (showFull) {
-            // Preview, favorite, settings
-            bottomSheetRoomPreviewItem {
-                id("room_preview")
-                avatarRenderer(host.avatarRenderer)
-                matrixItem(roomSummary.toMatrixItem())
-                stringProvider(host.stringProvider)
-                colorProvider(host.colorProvider)
-                izLowPriority(roomSummary.isLowPriority)
-                izFavorite(roomSummary.isFavorite)
-                settingsClickListener { host.listener?.didSelectMenuAction(RoomListQuickActionsSharedAction.Settings(roomSummary.roomId)) }
-                favoriteClickListener { host.listener?.didSelectMenuAction(RoomListQuickActionsSharedAction.Favorite(roomSummary.roomId)) }
-                lowPriorityClickListener { host.listener?.didSelectMenuAction(RoomListQuickActionsSharedAction.LowPriority(roomSummary.roomId)) }
-            }
+        // Preview, favorite, settings
+        bottomSheetRoomPreviewItem {
```

Comment:
Test `state.roomListActionsArgs.mode == RoomListActionsArgs.Mode.FULL` should stay

Reply:
Ah maybe not, it seems that only FULL is used nowadays.
Maybe cleanup https://github.com/vector-im/element-android/blob/main/vector/src/main/java/im/vector/app/features/home/room/list/actions/RoomListQuickActionsBottomSheet.kt#L50

Reply:
the previous condition was 

```kotlin
// V2 always shows full details as we no longer display the sheet from RoomProfile > Notifications
val showFull = state.roomListActionsArgs.mode == RoomListActionsArgs.Mode.FULL || isV2
```` 

now that v2 is always true, do we need the condition (perhaps it was wrong previously :thinking: )

Reply:
I do not think it was wrong before, it was OK in v1 mode, but [with this PR](https://github.com/vector-im/element-android/pull/4627/files#diff-0ed745b1f18ce801155cc8f6f7ff155ce1741a3c63e927a86fee0d46583bb425L265), `RoomListActionsArgs.Mode.NOTIFICATIONS` is never used, so maybe just delete `RoomListActionsArgs.Mode`.

Reply:
will do!

Reply:
removed c208c2d5ae9e6c6158756b654588eee096a57996